indico/modules/events/contributions/forms.py

```diff
@@ -32,3 +32,4 @@ class ContributionForm(IndicoForm):
     duration = TimeDeltaField(_("Duration"), [DataRequired()], default=timedelta(minutes=20),
                               units=('minutes', 'hours'),
                               description=_("The duration of the contribution"))
+    location_data = IndicoLocationField(_("Location"), description=_("Location"))
```

Comment:
you get a short "flash" of the message between the html being rendered and the js running. so unless you use css to hide it initially that's a bad idea